### ISSUE TITLE
[12.0] Create & Confirm SO (w/No Track Items With 0 Stock) >> Resulting Picking Forces Serial/Lot Wizard

### ISSUE THREAD

**Issue by lukebranch** [odoo/odoo#29474]
<h1>Impacted Versions:</h1>
12.0
<h1>Steps to Reproduce:</h1>
No demo, data, just clean install. Sales, Purchasing, Inventory, eCommerce, Invoicing.
<ul>
<li>Go to Settings > Inventory</li>
<li><img src="https://user-images.githubusercontent.com/2337666/49875677-91b7fa80-fe5c-11e8-9315-00dacb1523c1.png"/></li>
<li>Create a product of type 'Storeable'</li>
<li>Create an SO and add above product with Inventory on Hand QTY = 0 (ie. do not introduce stock to the system)</li>
<li>Confirm SO</li>
<li>Go to Picking</li>
<li>
<img src="https://user-images.githubusercontent.com/2337666/49875848-f8d5af00-fe5c-11e8-9ffa-0f2ca33dacfd.png"/>
</li>
<li>Try to add a done QTY in edit mode, or try to click 'Validate'</li>
<h1>Current Behaviour:</h1>
Lot/Serial Number wizard is forced, and no line is pre-filled.

<h1>Expected Behaviour:</h1>
No Lot/Serial Number icon (especially when not configured to track by lot/serial).<br/>
No forced Lot/Serial Number wizard.

**Comment by lukebranch** [comment]
@Yenthe666 ,

A small bug (at least as far as I can tell) that I ran into this one today. Latest code has been pulled for the 12.0 branch already.

Does not appear to be reproduce-able in runbot due to demo data.

**Comment by icoro** [comment]
Any updates on this? It has been more than 3 weeks since reported.

**Comment by Yenthe666** [comment]
@icoro no as long as there is no comment/feedback there is no update. The bug tracker here is a best effort. If you have it too and if you have an enterprise license you can make an official bug report at odoo.com/help (which gets a higher priority)

**Comment by icoro** [comment]
Yes, we have enterprise licenses. I will report it there. If we change the code and fix it, what is the process of adding the fix into the main source code?

**Comment by Yenthe666** [comment]
Awesome. If you have a working fix you can make a PR here. If the PR is approved by somebody at Odoo it'll be merged in basically.
